VIMUpload/UploadTaskQueue/Subtasks/VIMCreateTicketTask.m

```diff
@@ -271,6 +271,8 @@ - (void)sessionManager:(VIMNetworkTaskSessionManager *)sessionManager taskDidCom
         return;
     }
 
+    // TODO: Why do we not extract these upfront? [AH] 9/7/2015
+    
```

Comment:
We store the response dictionary upfront because we have to persist it. But just wondering why not extract the pertinent values upfront instead. To mitigate chance of losing that data along the way (seeing some errors in localytics related to this data having gone missing).